bigquery/google/cloud/bigquery/table.py

```diff
@@ -808,18 +824,38 @@ def items(self):
         return items
```

Comment:
Another possible re-factoring:

```python
for key, index in six.iteritems(self._xxx_field_to_index):
    yield (key, self._xxx_values[index])
```

Reply:
Done.

Ideally we'd somehow make this a "view" like in Python 3 dictionaries, but that seems like a lot of effort, especially since rows should be immutable.